build.gradle.kts

```diff
@@ -39,7 +37,7 @@ allprojects {
     
     tasks.withType<KotlinCompile> {
         kotlinOptions.apply {
-            languageVersion = "1.3"
+            languageVersion = "1.5" // must be the version that is bundled into the current Gradle
```

Comment:
It seems like we have to stick with `1.4` LV for a while (you might need `-Xsuppress-version-warnings` to pass through `allWarningsAsErrors` though).

Otherwise it won't be actually backwards compatible and any project that is using older version of gradle will get something like "Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.5.0, expected version is 1.4.1"

Reply:
Yes, it would be good if Gradle 6 (or even lower) would still be supported.